sim/validator.py

```diff
@@ -152,8 +153,8 @@ def performance_test(results_path, btb_seq_path, reference_path, exist_ok=False,
     # Analyse Results
     # HACK: this could easily break if additioanl files are present
     pipeline_directory = glob.glob(btb_seq_results_path + 'Results_simulated-reads_*')[0] + '/'
-    pipeline_snps = pipeline_directory + 'snpTables/simulated.tab'
-    stats = analyse(simulated_snps, pipeline_snps)
+    pipeline_snps = pipeline_directory + 'snpTables/simulated_snps.tab'
```

Comment:
"simulated_snps.tab" is inconsitent with the prefix ("simulated") set on line 55. Therefore analyse() can't find this file.

Reply:
Actually this is correct! Richard changed the naming convention of the SNPs .tab table to have a _snps.tab postfix with his latest `btb-seq` PR. This is how it used to work, and downstream processes (like LIMS and Richard's experimental phylogeny tools), requires this formatting.

Reply:
Ok, but then shouldn't line 55 be changed to `output_prefix = output_path + "simulated_snps"?`

Reply:
Ah ok, my mistake. I sse now.